lambda/db/src/migrations/2022.05.17T10.30.00.migrate-clientname-to-clientid.ts

```diff
@@ -0,0 +1,36 @@
+import { DataTypes, Op } from 'sequelize';
+
+export const name = '2022.05.17T10.30.00.migrate-clientname-to-clientid';
+
+export const up = async ({ context: sequelize }) => {
+  console.log('sequelize', Object.keys(sequelize));
```

Comment:
Do we want this console log staying here?

Reply:
good catch, I removed it.